src/app/views/sidebar/sample-queries/SampleQueries.tsx

```diff
@@ -59,10 +65,14 @@ export class SampleQueries extends Component<ISampleQueriesProps, any> {
     } else {
       this.props.actions!.fetchSamples();
     }
+    const { sampleQueries } = this.state;
+    this.groups = generateGroupsFromList(sampleQueries, 'category');
```

Comment:
why was this added? when i remove these lines it still works

Reply:
It seems to regenerate the groups every render so adding this prevented the group from defaulting the collapse every time someone selected a query.